src/app/shared/palette/order-summary/write-order-summary-field.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit } from '@angular/core';
+import { AbstractFieldWriteComponent } from '../base-field/abstract-field-write.component';
+import { FormControl, FormGroup, FormArray } from '@angular/forms';
+
+@Component({
+  selector: 'ccd-write-order-summary-field',
+  templateUrl: './write-order-summary-field.html'
+})
+export class WriteOrderSummaryFieldComponent extends AbstractFieldWriteComponent implements OnInit {
+
+  paymentReference: FormControl;
+  paymentTotal: FormControl;
+  feesArray: FormArray;
+  orderSummaryGroup: FormGroup;
+
+  ngOnInit(): void {
+    this.orderSummaryGroup = this.registerControl(new FormGroup({}));
```

Comment:
Could a comment be added just to explain quickly why those are implemented manually rather than making use of `WriteComplexFieldComponent`? Basically, that given the view is read-only, the tree of form controls is not being built automatically and need to be built manually instead.